infrastructure/lambda/template.yaml

```diff
@@ -778,50 +869,38 @@ Resources:
 
 Outputs:
 
-  SessionApiUrl:
-    Description: "URL for the Address API /session resource"
-    Value: !Sub "https://${AddressApi}.execute-api.${AWS::Region}.amazonaws.com/${Environment}/session"
-    Export:
-      Name: !Sub ${AWS::StackName}-SessionApiUrl
-
-  PostcodeLookupApiUrl:
-    Description: "URL for the Address API /postcode-lookup resource"
-    Value: !Sub "https://${AddressApi}.execute-api.${AWS::Region}.amazonaws.com/${Environment}/postcode-lookup"
-    Export:
-      Name: !Sub ${AWS::StackName}-PostcodeLookupApiUrl
-
-  AddressApiUrl:
-    Description: "URL for the Address API /address resource"
-    Value: !Sub "https://${AddressApi}.execute-api.${AWS::Region}.amazonaws.com/${Environment}/address"
+  AddressApiBaseUrl:
```

Comment:
would rename this with `Public` in the name, or at least add a second one with `Public` for now, and then moves things (dns/FE) over to the new one.

Reply:
Just saw that is exactly what you did :)